File: src/main/java/um/haberes/report/model/haberes/core/AnotadorDto.java
package um.haberes.report.model.haberes.core;

import java.time.LocalDateTime;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AnotadorDto {

    private Long anotadorId;

    private Long legajoId;

    private int anho = 0;

    private int mes = 0;

    private Integer facultadId;

    private String anotacion;

    private byte visado = 0;

    private String ipVisado;

    private String user;

    private String respuesta;

    private byte autorizado = 0;

    private byte rechazado = 0;

    private byte rectorado = 0;

    private byte transferido = 0;

    private PersonaDto persona;

    private FacultadDto facultad;

    private LocalDateTime created;

}

File: src/main/java/um/haberes/report/kotlin/dto/haberes/core/AnotadorDto.kt
package um.haberes.report.kotlin.dto.haberes.core

import java.time.LocalDateTime

data class AnotadorDto(

    var anotadorId: Long? = null,
    var legajoId: Long? = null,
    var anho: Int = 0,
    var mes: Int = 0,
    var facultadId: Int? = null,
    var anotacion: String? = null,
    var visado: Byte = 0,
    var ipVisado: String? = null,
    var user: String? = null,
    var respuesta: String? = null,
    var autorizado: Byte = 0,
    var rechazado: Byte = 0,
    var rectorado: Byte = 0,
    var transferido: Byte = 0,
    var persona: PersonaDto? = null,
    var facultad: FacultadDto? = null,
    var created: LocalDateTime? = null

)
